src/clarity-angular/popover/signpost/signpost-content.ts

```diff
@@ -44,10 +44,10 @@ export class SignpostContent/* implements OnDestroy, AfterViewInit*/ {
      */
     close() {
         // This can only be called when a signpost content is open
-        this.signpost.toggleSignpost();
+        this.ifOpenService.open = !this.ifOpenService.open;
```

Comment:
Then you might as well just `this.ifOpenService.open = false`, right? 😛  Easier to read. (Yes, this is a nitpick.)